<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinSettings</class>
 <widget class="QMainWindow" name="WinSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_settings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>General </string>
     </attribute>
     <widget class="QLineEdit" name="line_bench_dir">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>431</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search_bench_dir">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="index" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_bench_dir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Benchmark-suite's directory: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_base_dir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Base test directory:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_base_dir">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>431</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search_base_dir">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="index" stdset="0">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_base_dir_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>where files are read from and written to</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Advanced</string>
     </attribute>
     <widget class="QLineEdit" name="txt_test_time">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_max_startup">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_max_rate">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_sync">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_test_time">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Test time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_test_time_desc">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Used by aggregate throuput test</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_start">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max startup time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_start_desc">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Used by common startup time test</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_rate">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max rate:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_rate_desc">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min value is : Max value is:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_sync">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sync:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="slc_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_custom_w">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>W</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rw_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Read/Write type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_custom_r">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QComboBox" name="slc_1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>slc_command</string>
      </property>
      <item>
       <property name="text">
        <string>gnome-terminal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>xterm</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="slc_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>slc_n_iteration</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_num_rw_custom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>N° Reads &amp; Writes Custom:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_command">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Command:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_n_iteration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>112</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>N° Iterations:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="slc_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>slc_rw_type</string>
      </property>
      <item>
       <property name="text">
        <string>seq</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rand</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>raw_seq</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>raw_rand</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="slc_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_file_size">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_file_size">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>File size</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_file_size_desc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Size of file to be copied</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_default">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>440</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
